aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -6,14 +6,17 @@ module aptos_framework::staking_config {
 
     friend aptos_framework::genesis;
 
-    /// Stake lockup duration cannot be zero
+    /// Stake lockup duration cannot be zero.
     const EZERO_LOCKUP_DURATION: u64 = 1;
-    /// Reward rate denominator cannot be zero
+    /// Reward rate denominator cannot be zero.
     const EZERO_REWARDS_RATE_DENOMINATOR: u64 = 2;
-    /// Specified stake range is invalid. Max must be greater than min
+    /// Specified stake range is invalid. Max must be greater than min.
     const EINVALID_STAKE_RANGE: u64 = 3;
-    /// The voting power increase limit percentage must be within (0, 50]
+    /// The voting power increase limit percentage must be within (0, 50].
     const EINVALID_VOTING_POWER_INCREASE_LIMIT: u64 = 4;
+    /// Specified rewards rate is invalid, which must be within [0, 100).
+    const EINVALID_REWARDS_RATE: u64 = 5;
```

Comment:
May be EREWARDS_RATE_TOO_HIGH to be more specific? The comment can say rewards rate must be smaller than 100%